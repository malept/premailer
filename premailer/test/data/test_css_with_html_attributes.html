<html>
<head>
<style type="text/css">
td { background-color:red; }
p { text-align:center; }
table { width:200px; }
</style>
</head>
<body>
<p>Text</p>
<table>
  <tr>
    <td>Cell 1</td>
    <td>Cell 2</td>
  </tr>
</table>
</body>
</html>
