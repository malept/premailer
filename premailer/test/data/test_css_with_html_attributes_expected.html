<html>
<head>
</head>
<body>
<p style="text-align:center" align="center">Text</p>
<table style="width:200px" width="200">
  <tr>
    <td style="background-color:red" bgcolor="red">Cell 1</td>
    <td style="background-color:red" bgcolor="red">Cell 2</td>
  </tr>
</table>
</body>
</html>
